sigs.yaml

```diff
@@ -678,20 +678,10 @@ sigs:
       slack: sig-cluster-lifecycle
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle
       teams:
-      - name: sig-cluster-lifecycle-api-reviews
-        description: API Changes and Reviews
-      - name: sig-cluster-lifecycle-bugs
-        description: Bug Triage and Troubleshooting
-      - name: sig-cluster-lifecycle-feature-requests
-        description: Feature Requests
-      - name: sig-cluster-lifecycle-misc
-        description: General Discussion
+      - name: sig-cluster-lifecycle
+        description: Notify group
       - name: sig-cluster-lifecycle-pr-reviews
```

Comment:
Aren't we going to delete the pr-reviews one too?

Reply:
I'm following the pattern that we might want to separate reviews from global notifications.